jbpm-wb-integration/jbpm-wb-integration-client/src/main/java/org/jbpm/workbench/wi/client/editors/deployment/descriptor/items/NamedObjectItemPresenter.java

```diff
@@ -111,5 +131,9 @@ public void signalParameterAddedOrRemoved() {
         void setValue(final String value);
 
         void setParametersCount(final int size);
+
+        HTMLInputElement getNameElement();
```

Comment:
Please avoid manipulating view elements in presenter logic.

Reply:
yes you are right, I updated it .Please review again.